source/change-streams/tests/unified/change-streams.yml

```diff
@@ -8,6 +8,12 @@ createEntities:
       id: &client0 client0
       observeEvents:
         - commandStartedEvent
+      ignoreCommandMonitoringEvents:
+        - killCursors
+      useMultipleMongoses: false
+  - client:
+      id: &globalClient globalClient
```

Comment:
Per my comments above, I'll note that none of the existing tests in this file required global clients, databases, or collections.

That said, if these tests will operate on multiple collections you may want to explicitly initialize those in `initialData` (as I did in [`poc-change-streams.yml`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/tests/valid-pass/poc-change-streams.yml#L47)).